src/main/connector/system/WindowsProcess/WindowsProcess.yaml

```
@@ -0,0 +1,68 @@
+extends:
+- ../System/System
+connector:
+  displayName: WindowsProcess
+  platforms: Any platform with Windows
+  reliesOn: WMI
+  information: Gives OS specific information and metrics
+  detection:
+    connectionTypes:
+    - remote
+    - local
+    appliesTo:
+    - nt
+    criteria:
+    - type: wmi
+      namespace: root\CIMv2
+      query: SELECT * FROM Win32_OperatingSystem

```

```suggestion
      query: SELECT Name FROM Win32_OperatingSystem
```

No need to query all fields and trigger unnecessary computation on the monitored system (and in our WMI stack).